### PR TITLE
Add template-prometheus-mixin command

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Add get-jsonnet command

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
I added the env to introduce some flexibility in case someone wants to generate something from local directory

File: README.md [hunk 1]
```diff
@@ -164,6 +164,23 @@ You can add additional parameters (like `--version=1.0.0`) at the end of the com
...
+`get-jsonnet` is a command which generates `remoteWrite` configuration for the kube prometheus.
+
+```
+```
+ kubectl run template-dependency \
+  -it --quiet --rm \
+  --restart=Never -n sumologic \
+  --image sumologic/kubernetes-tools \
+  -- get-libjsonnet > kube-prometheus-sumo-logic-mixin.libsonnet
+```
```
Comment (pmalek-sumo) [inline README.md:178]: Lol the suggestions breaks around backticks but there's one superfluous line with 3 backticks (can't even type them here 🤷🏻 )

File: README.md [hunk 2]
```diff
@@ -164,6 +164,23 @@ You can add additional parameters (like `--version=1.0.0`) at the end of the com
 List of supported arguments is compatible with
 [`helm show values`](https://helm.sh/docs/helm/helm_show_values/).
 
+### Jsonnet prometheus-operator configuration
+
+`get-jsonnet` is a command which generates `remoteWrite` configuration for the kube prometheus.
```
Comment (pmalek-sumo) [inline README.md:169]:
```suggestion
`get-jsonnet` is a command which generates `remoteWrite` configuration for kube prometheus.
```

File: src/commands/template-dependency [hunk 3]
```diff
@@ -11,6 +11,8 @@ if [[ "${arg}" == "--help" ]]; then
   exit 0
 fi
 
+REPO_PATH="${REPO_PATH:-sumologic/sumologic}"
```
Comment (pmalek-sumo) [inline src/commands/template-dependency:14]: `readonly` ? 🤷🏻

File: src/commands/template [hunk 4]
```diff
@@ -11,6 +11,8 @@ if [[ "${arg}" == "--help" ]]; then
   exit 0
 fi
 
+REPO_PATH="${REPO_PATH:-sumologic/sumologic}"
```
Comment (pmalek-sumo) [inline src/commands/template:14]: `readonly` ? 🤷🏻

**Review by perk-sumo — APPROVED** [review]
👍

**Pull request merged**